src/PKSim.UI.Starter/SimulationTransferConstructor.cs

```diff
@@ -1,17 +1,29 @@
-﻿using PKSim.Core.Services;
+﻿using OSPSuite.Core.Qualification;
+using PKSim.CLI.Core.Services;
+using PKSim.Core.Services;
 
-namespace PKSim.UI.Starter
+namespace PKSim.UI.Starter;
+
+public static class SimulationTransferConstructor
 {
-   public static class SimulationTransferConstructor
+   public static object CreateSimulationTransfer(string projectSnapshot)
+   {
+      var container = CLIApplicationStartup.Initialize();
+      var projectSnapshotToSimulationTransferMapper = container.Resolve<IProjectSnapshotToSimulationTransferMapper>();
+
+      return projectSnapshotToSimulationTransferMapper.MapFrom(projectSnapshot).transfer;
+   }
+
+   public static object CreateSimulationTransferAndExportInputs(string projectSnapshot, QualificationConfiguration qualificationConfiguration)
    {
-      public static object CreateSimulationTransfer(string projectSnapshot)
-      {
-         var container = ApplicationStartup.Initialize();
-         var projectSnapshotToSimulationTransferMapper = container.Resolve<IProjectSnapshotToSimulationTransferMapper>();
-         var moBiExportTask = container.Resolve<IMoBiExportTask>();
+      var container = CLIApplicationStartup.Initialize();
+      var projectSnapshotToSimulationTransferMapper = container.Resolve<IProjectSnapshotToSimulationTransferMapper>();
+      var qualificationInputTask = container.Resolve<IQualificationInputTask>();
+
+      var (simulationTransfer, project) = projectSnapshotToSimulationTransferMapper.MapFrom(projectSnapshot);
```

Comment:
in a project snapshot there is only ever one simulation?

Reply:
In this case yes, because it's coming from a Module. PK-Sim builds the module for `SimulationTransfer` - only one simulation.